src/particle/pdgid/__init__.py

```diff
@@ -61,6 +61,15 @@
 
 from __future__ import absolute_import
 
+from typing import Tuple
+
+from .functions import is_composite_quark_or_lepton  # noqa: F401
```

Comment:
All these should be imported normally as all others, no noqa.

Reply:
Then they need to be added to `__all__`, they were missing before.